taverna-docker-activity/src/test/java/org/apache/taverna/activities/docker/test/TestDockerCommands.java

```diff
@@ -40,7 +40,7 @@
 
     private static final String DOCKER_LOGIN_SUCCESS = "Login Succeeded";
 
-    public TestDockerCommands(){
+    public TestDockerCommands() {
         DockerContainerConfigurationImpl containerConfiguration = new DockerContainerConfigurationImpl(new TestConfigurationManager());
         containerConfiguration.getInternalPropertyMap().put(DockerContainerConfigurationImpl.NAME,CONTAINER_NAME);
```

Comment:
containerConfiguration should be a DockerContainerConfiguration. Any code specific to the implementation should not be visible